src/main/java/spoon/support/sniper/internal/ElementSourceFragment.java

```diff
@@ -720,7 +721,7 @@ private void onCharSequence(CharType type, StringBuilder buff, Consumer<SourceFr
 			}
 			// nothing has matched, best effort token
 			if (longestMatcher == null) {
-				consumer.accept(new TokenSourceFragment(str.toString(), TokenType.CODE_SNIPPET));
+				consumer.accept(new TokenSourceFragment(Arrays.toString(str), TokenType.CODE_SNIPPET));
```

Comment:
very confusing to have an array named str, `charArray` is better?

Reply:
Yes, i renamed it